### DISCUSSION TITLE
Will other indexes be added with NHANES before 2005?

### DISCUSSION THREAD

**Discussion by mkou0019** [jamesjiadazhan/dietaryindex#286] [Q&A] [answered]
Hi, thank you for the package! 

Do authors have plans to add functions using NHANES data before 2005? like HEI2015_NHANES_MPED() and DII_NHANES_MPED()?

**Comment by jamesjiadazhan** [comment]
Thanks for asking! Not recent plan, but may do that in the next few months.

**Comment by jamesjiadazhan** [comment, marked as answer]
More dietary indexes, HEI2020, AHEI, DASH, DASHI, MED, MEDI are available for NHANES before 2005! Download the most recent package to enjoy the updated functions.
